app/assets/javascripts/client/inventories/InviteInventoryUserCtrl.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient').controller('InviteInventoryUserCtrl', InviteInventoryUserCtrl);
+
+  InviteInventoryUserCtrl.$inject = ['$scope', 'InventoryInvitation'];
+  function InviteInventoryUserCtrl($scope, InventoryInvitation) {
+    var vm = this;
+    vm.sendInvitation = function(invitation) {
+      $scope.alerts  = [];
+      $scope.addAlert = function(message) {
```

Comment:
For this, we really should be using `vm` as opposed to nesting this inside of this function.